README.md

```diff
@@ -59,30 +60,31 @@ Introduction to MoltenVK
 
 **MoltenVK** is a layered implementation of [*Vulkan 1.0*](https://www.khronos.org/vulkan) 
 graphics and compute functionality, that is built on Apple's [*Metal*](https://developer.apple.com/metal) 
-graphics and compute framework on both *iOS* and *macOS*. **MoltenVK** allows you to use *Vulkan* graphics 
-and compute functionality to develop modern, cross-platform, high-performance graphical games and applications, 
-and to run them across many platforms, including both *iOS* and *macOS*.
+graphics and compute framework on *macOS*, *iOS*, and *tvOS*. **MoltenVK** allows you to use *Vulkan* 
+graphics and compute functionality to develop modern, cross-platform, high-performance graphical 
+games and applications, and to run them across many platforms, including *macOS*, *iOS*, and *tvOS*.
 
 *Metal* uses a different shading language, the *Metal Shading Language (MSL)*, than 
 *Vulkan*, which uses *SPIR-V*. **MoltenVK** automatically converts your *SPIR-V* shaders 
 to their *MSL* equivalents.
 
-To provide *Vulkan* capability to the *iOS* and *macOS* platforms, **MoltenVK** uses *Apple's* 
+To provide *Vulkan* capability to the *macOS*, *iOS*, and *tvOS* platforms, **MoltenVK** uses *Apple's* 
 publicly available API's, including *Metal*. **MoltenVK** does **_not_** use any private or
 undocumented API calls or features, so your app will be compatible with all standard distribution 
 channels, including *Apple's App Store*.
 
 The **MoltenVK** runtime package contains two products:
 
-- **MoltenVK** is a driver-level implementation of the [*Vulkan 1.0*](https://www.khronos.org/vulkan) 
-  graphics and compute API.
+- **MoltenVK** is a implementation of an almost-complete subset of the 
+  [*Vulkan 1.0*](https://www.khronos.org/vulkan) graphics and compute API.
```

Comment:
Well, this should make Neil happy. 😃

Reply:
Ha! Actually...the Vulkan WG. But yes...it was inspired by the feedback we've received. 😉